website/src/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: home
+hero:
+  name: Task
+  text: The Modern Task Runner
+  tagline: A fast, cross-platform build tool inspired by Make, designed for modern workflows.
+  image:
+    src: /img/logo.png
+    alt: Task logo
+  actions:
+    - theme: brand
+      text: Get Started
+      link: /docs/getting-started
+    - theme: alt
+      text: Schema
+      link: /docs/reference/schema
+    - theme: alt
+      text: CLI
+      link: /docs/reference/cli
+
```

Comment:
These point to `/docs/*` but the pages are not under `/docs`. Somehow the link still works though?

Either way, I think these _should_ be hosted under `/docs` like they are today. It means we don't have to set up any redirect rules and keeps doc pages logically grouped under a URL path.

Reply:
Oopsy the URL is not correct. it should be `/reference/cli`

I'd love have docs but in our current setup it's not : 
- https://taskfile.dev/reference/cli
- https://taskfile.dev/usage/